main.cpp

```diff
@@ -108,7 +108,17 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
-
+// I started this one but got stuck because I don't really understand what the question is asking
+struct Person
+{
+    int age;
+    int height;
+    float hairLength;
+    float GPA;
+    unsigned int SATScore;
+    
+    void run(int howFast, bool startWithLeftFoot);
+};
```

Comment:
The question is asking you to COPY from the video the implementation of `run`, and then you figure out the definition for the type that `leftFoot` and `rightFoot` are, based on how they're used in the `run` function.
- [ ] revise

Reply:
I studied a few of the other students work to figure this one out.  I think I am starting to understand, although some parts are still confusing. 

- in `leftFoot.stepForward` is `stepForward` happening to `leftFoot`?
- Why are we declaring the functions inside the struct and then defining them outside?